src/main/kotlin/cacophony/codegen/patterns/CacophonyPattern.kt

```diff
@@ -1,22 +1,215 @@
 package cacophony.codegen.patterns
 
 import cacophony.codegen.instructions.Instruction
-import cacophony.controlflow.CFGNode
-import cacophony.controlflow.Register
-import cacophony.controlflow.RegisterLabel
+import cacophony.controlflow.*
 
 class CacophonyPattern {
+    /*****************************
+     * CHAPTER 1. VALUE PATTERNS *
```

Comment:
Maybe we can split these patterns into separate files? They don't really have to belong to `CacophonyPattern` class.

There's going to be a lot of them, so imo we can start organizing this already.

Reply:
We can use marker interfaces to group them semantically as well, it might help in the future